CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## Unreleased
+
+## Bootes - 16-08-2019
+### Added
+- New plugin architecture for the following Asperathos components:
+   - Monitor
+   - Manager
+   - Visualizer
+   - Controller
+- Persistent plugins for the following Asperathos components:
+   - Monitor
+   - Manager
+   - Visualizer
+   - Controller
+- PID scalling strategy at Asperathos Controller 
+
+- New tests for kubejobs plugin at Asperathos Monitor
+
+### Changed
+- Refactoring of the kubejobs plugin at Asperathos Manager
```

Comment:
Highlight ```kubejobs```. Rephrase to "Refactoring of the kubejobs plugin of the Manager component."